### PR TITLE
account  auto-index-creation env addition

### PR DESCRIPTION
**Pull request by denduluriramasaimohanakhil**
account  auto-index-creation env addition

### CODE REVIEW

File: tp-app-payment-gateway-dev.yaml [hunk 1]
```diff
@@ -23,6 +23,15 @@ paymentgateway:
     keysecret: ${PAYPAL_KEY_SECRET:EMuFpv1mkjSavIVveEdq3zxLMjuDl8SOPYLjl8dgtRhDqI2zXK-YT3l95jpPbk-uS1mFmyM6r1RnTvc5}
     bncode: ${PAYPAL_BN_CODE:Creed_SP_PPCP}
   payby:
+
```
Comment (VaibhavTechsophy) [inline tp-app-payment-gateway-dev.yaml:26]: why is this modified?

**Review by VaibhavTechsophy — CHANGES_REQUESTED** [review]
remove dollar variable

File: tp-app-payment-gateway-dev.yaml (hunk 1 above)
Comment (denduluriramasaimohanakhil) [reply tp-app-payment-gateway-dev.yaml:26]: i did not modify this payment gateway vaibhav they are from other branch

**Comment by VaibhavTechsophy** [comment]
open new merge request with correct names
